packages/create-app/template-vue-ts/src/components/HelloWorld.vue

```diff
@@ -1,49 +1,65 @@
 <template>
   <h1>{{ msg }}</h1>
+  <label>
+    <input type="checkbox" :checked="useScriptSetup" @input="setUseScriptSetup" /> Use
+    <code>&lt;script setup&gt;</code>
+  </label>
+  <label> <input type="checkbox" :checked="useTsPlugin" @input="setUseTsPlugin" /> Provide types for <code>*.vue</code> imports </label>
 
   <p>
     Recommended IDE setup:
     <a href="https://code.visualstudio.com/" target="_blank">VSCode</a>
     +
-    <a
-      href="https://marketplace.visualstudio.com/items?itemName=octref.vetur"
-      target="_blank"
-    >Vetur</a>
-    or
-    <a href="https://github.com/johnsoncodehk/volar" target="_blank">Volar</a>
-    (if using
-    <code>&ltscript setup&gt;</code>)
+    <template v-if="!useScriptSetup">
+      <a href="https://marketplace.visualstudio.com/items?itemName=octref.vetur" target="_blank">Vetur</a>
+      +
+      <a href="https://marketplace.visualstudio.com/items?itemName=znck.vue-language-features" target="_blank"
+        >Vue DX</a
+      >
```

Comment:
Line breaks look odd to me.

```suggestion
      <a
        href="https://marketplace.visualstudio.com/items?itemName=znck.vue-language-features"
        target="_blank"
      >Vue DX</a>
```

Reply:
Looks odd to me too. Fixed.

The issue I raised was temporarily fixed by just removing all of the offending code. I would hate for that to become a permanent "fix" when I found how to fix it... But it no longer affects me so up to you 😊